integration-tests/tests/janitor.js

```diff
@@ -27,6 +27,8 @@ const runJanitor = withUserToken(async ({ billingProject, page, testUrl, token }
 
   return Promise.all(_.map(async ({ workspace: { namespace, name } }) => {
     try {
+      // Unlock the workspace in case it was left in a locked state during a test (locked workspaces can't be deleted).
```

Comment:
We ended up with some locked workspaces due to the test failing before unlocking again. Might as well make sure workspaces are unlocked before we try to delete them.